static/app/views/dashboards/contexts/widgetSyncContext.stories.tsx

```diff
@@ -34,14 +34,14 @@ export default storyBook('WidgetSyncContext', story => {
             <MediumWidget>
               <LineChartWidget
                 title="span.duration"
-                timeseries={[sampleDurationTimeSeries]}
+                timeSeries={[sampleDurationTimeSeries]}
```

Comment:
I dont want to open a new can of worms, but why the time prefix? Are the charts limited to a time axis?

Reply:
Two reasons!

1. A "time series" is common parlance, whereas a "series" is more of an ECharts concept, and I want to keep them separate
2. Yep, Line, Bar, and Area charts right now _only_ support time series, rather than generic series. I will need to open up bar charts soon, though

Reply:
I never thought series is an echarts thing, it just kind of made sense to me to name it that way.

If the charts only support time series, then that naming makes sense to me, with the tiny caveat that the LineChartWidget naming doesn't convey that restriction well. Anyways, topic for some other time or PR :)

Reply:
👍🏻 fair